src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Apprentice/DetailsViewModel.cs

```diff
@@ -63,10 +63,13 @@ public class DetailsViewModel : IAuthorizationContextModel
         public string EmploymentPriceDisplay => EmploymentPrice?.ToGdsCostFormat() ?? string.Empty;
         public DateTime? EmploymentEndDate { get; set; }
         public string EmploymentEndDateDisplay => EmploymentEndDate?.ToGdsFormatWithoutDay() ?? string.Empty;
-        public bool RecognisePriorLearning { get; set; }
-        public int DurationReducedBy { get; set; }
+        public bool? RecognisePriorLearning { get; set; }
+        public int? TrainingTotalHours { get; set; }
+        public int? DurationReducedBy { get; set; }
         public string DurationReducedByDisplay => $"{DurationReducedBy} weeks";
```

Comment:
@Pauljgraham I like this, but is it necessary? Should we just use the view (cshtml) for textual concerns